addon/components/date-picker/mobile/template.hbs

```diff
@@ -9,15 +9,15 @@
 </div>
 
 <div class="date-picker-mobile-days month-{{currentMonth.month}}">
-  {{#horizontal-list-swiper/sly
-    itemsUpdate=days
-    indexUpdate=currentIndex
-    active-item-alignment="forceCentered"
-    onmove=(action "onmove")
-    onmoveend=(action "onmoveend")
-    cacheKey=cacheKey
+  <HorizontalListSwiper::Sly
+    @itemsUpdate={{days}}
```

Comment:
{{this.days}} anyway it will deprecated only in 4+. So not blocking this PR though. 
But do you want to merge it direct to master?

Reply:
fixed. Yes should be done everywhere, not just in angle bracket components. The ember-octane branch has linting for those I think

Reply:
Merging to master as tests are passing - can't QA test it until we upgrade the apps so I guess can still do releases and add patches later on if needed

Reply:
@Leooo Test failed. I didn't know we still use Travis.

Reply:
@somasuna should be fixed now. Travis is used here because it's an open-sourced addon.

Reply:
> Merging to master as tests are passing - can't QA test it until we upgrade the apps so I guess can still do releases and add patches later on if needed

@Leooo I would still say we have to treat it same way as our Ember commons addon, and follow our standard process feature -> epic -> master. Also it has to be tested in one our apps.

Reply:
@somasuna then we'll have to wait 3 months or so for it to be consumable by any of our apps internally. And I'm keen to update up to Octane before that, so that we can share / showcase it with the Ember community, that could help solve bugs before we consume it etc. Keep in mind it's an open-source addon, not an internal one.

Reply:
we dont have to wait to test but should follow standard process branching strategy wise